sdk_extensions/zpages/src/test/java/io/opentelemetry/sdk/extensions/zpages/ZPageServerTest.java

```diff
@@ -18,9 +19,16 @@
   void tracezSpanProcessorOnlyAddedOnce() throws IOException {
     // tracezSpanProcessor is not added yet
     assertThat(ZPageServer.getIsTracezSpanProcesserAdded()).isFalse();
-    HttpServer server = HttpServer.create(new InetSocketAddress(8888), 5);
-    ZPageServer.registerAllPagesToHttpServer(server);
-    // tracezSpanProcessor is added
-    assertThat(ZPageServer.getIsTracezSpanProcesserAdded()).isTrue();
+    HttpServer server = null;
+    try {
+      server = HttpServer.create(new InetSocketAddress(new Random().nextInt(1000) + 2000), 5);
```

Comment:
Doesn't port `0` work?

Reply:
hmm. good question. from the javadocs, yes! good catch!

Reply:
fixed.